### DISCUSSION TITLE
Image utils in the SDK

### DISCUSSION THREAD

**Discussion by AnishDe12020** [directus/directus#13071] [Feature Requests]
Currently, if I request a file through the SDK, I am returned the UUID and if I request the specific fields, I get all the fields except for a URL to the image. Obviously, it is quite easy to construct one but I think it would be a great feature to have some in-built features in the SDK if passed in an option (say `generateImage`), then it will generate the image URL and return that as well.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
